banzai/bpm.py

```diff
@@ -17,7 +17,7 @@ def do_stage(self, images):
         for image in images:
             add_bpm_to_image(image, self.pipeline_context)
             validate_bpm_size(image)
-            if image.header['L1IDMASK'][0] == '' and not self.pipeline_context.no_bpm:
+            if image.header.get('L1IDMASK') == '' and not self.pipeline_context.no_bpm:
```

Comment:
I would update to image.header.get('L1IDMASK', '')

Reply:
Good idea, done